### DISCUSSION TITLE
What does CAPTCHA stand for?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#898] [Galaxy Trivia] [answered]
**Bonus:** What is the human error rate for their completion?

**Comment by peterbaccaro** [comment, marked as answer]
CAPTCHA stands for "**Completely Automated Public Turing test to tell Computers and Humans Apart**".  

CAPTCHAs Have an 8% Failure Rate, and 29% if Case Sensitive.
